thavalon-webapp/src/App.tsx

```diff
@@ -46,7 +50,13 @@ function App() {
 
   return (
     <div>
-      <Navbar loggedIn={loggedIn} useMobileMenu={useMobileMenu} setUseMobileMenu={setUseMobileMenu} setShowLoginModal={setShowLoginModal} />
+      <Navbar
+        loggedIn={loggedIn}
+        useMobileMenu={useMobileMenu}
+        setUseMobileMenu={setUseMobileMenu}
+        setShowLoginModal={setShowLoginModal}
+        setShowCreateJoinGameModal={setShowCreateJoinGameModal}
```

Comment:
Out of scope, but I wonder if we could avoid keeping track of all this state by putting modals on their own pages and using normal react-router `<Route />`s. The pages could render just the modal, and then navigate to the home page if it's closed. I don't quite remember, but this might also be reopening an old can of worms

Reply:
Could make an issue for this for tracking?